packages/core/package.nls.json

```diff
@@ -27,6 +27,7 @@
     "AWS.stepFunctions.workflowStudio.actions.progressMessage": "Opening asl file in Workflow Studio",
     "AWS.stepFunctions.workflowStudio.actions.saveSuccessMessage": "{0} has been saved",
     "AWS.stepFunctions.workflowStudio.actions.invalidJson": "The Workflow Studio editor was not opened because the JSON in the file is invalid. To access Workflow Studio, please fix the JSON and manually reopen the integration.",
+    "AWS.stepFunctions.workflowStudio.enable.desc": "Open Amazon States Language files in Workflow Studio by default.",
```

Comment:
Use `workbench.editorAssociations`. We already do something similar for threat composer: https://github.com/aws/aws-toolkit-vscode/blob/30face99ec161876378b49e71e78ae3714ed5797/packages/toolkit/package.json#L4412-L4415

Do not add new, custom ways of setting file associations.